docs/reference/integrations.rst

```diff
@@ -108,6 +109,17 @@ You can override this behavior to use a native pydantic validation/serialization
 
 .. literalinclude:: /examples/reference/integrations/native_pydantic.py
 
+.. _msgspec:
+
+Msgspec
+=============
+
+By default, any msgspec Struct is loaded and dumped like any other model.
+For example, any unsupported by to_builtins types won't be casted.
```

Comment:
Why is to_builtins mentioned here when the previous sentence talks about behavior using the model directly?